module/importer/src/Application/DependencyInjection/CompilerPass/SourceTypeCompilerPass.php

```diff
@@ -37,14 +35,13 @@ public function process(ContainerBuilder $container): void
      */
     private function processServices(ContainerBuilder $container): void
     {
-        $arguments = [];
-        $definition = $container->findDefinition(SourceTypeDictionaryProvider::class);
-        $strategies = $container->findTaggedServiceIds(self::TAG);
-        $translator = $container->findDefinition(TranslatorInterface::class);
+        $definition = $container->findDefinition(SourceTypeProvider::class);
+        $services = $container->findTaggedServiceIds(self::TAG);
 
-        $arguments[] = $translator;
-        foreach ($strategies as $id => $strategy) {
-            $arguments[] = new Reference($id);
+        $arguments = [];
+        foreach ($services as $id => $service) {
+            $arguments[] = $id;
```

Comment:
I think the assumption that $id === FQCN is invalid. Please get the class from the service definition rather than the id.